### DISCUSSION TITLE
MeterValue not appeared in handler and can I send SetChargingProfile in starttransaction handler

### DISCUSSION THREAD

**Discussion by vcba79** [ChargeTimeEU/Java-OCA-OCPP#359] [Q&A]
Hello, 

I am trying to implement charger controller using Java-OCA-OCPP. 

Server start ok and accept Authorize, StatusNotification, etc but not MeterValues. I have a python ocpp echo server shows charger send MeterValues after transaction started. But server provied by this project never call MeterValues handler. How can I identify this issue and solve it?

Is there a code example to show how to send SetChargingProfile in  starttransaction handler? I have not idea how to send SetChargingProfile  after transaction started.

Thanks,

**Comment by jmluy** [comment]
@vcba79 The SetChargingProfile message can be sent similar to how you send other messages.

```
SetChargingProfileRequest request = new SetChargingProfileRequest(connectorId, chargingProfile);
jsonServer.send(sessionId, request);
```
